indexer/locality_object.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include "base/buffer_vector.hpp"
+#include "base/osm_id.hpp"
+
+#include <cstdint>
+#include <vector>
+
+namespace indexer
+{
+class LocalityObject
```

Comment:
Please add a short description to this class.

Reply:
done